<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Ph. Faist</author>
 <class>KLFLibBrowser</class>
 <widget class="QWidget" name="KLFLibBrowser">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KLatexFormula Library</string>
  </property>
  <property name="windowIcon">
   <iconset resource="klfres.qrc">
    <normaloff>:/pics/hi16-app-klatexformula.png</normaloff>:/pics/hi16-app-klatexformula.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QSplitter" name="splitViewsEditor">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frmViews">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="lyt_frmViews">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabResources">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_dummy">
          <attribute name="title">
           <string notr="true">Dummy Tab (that will be deleted at execution)</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="KLFLibEntryEditor" name="wEntryEditor" native="true"/>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frmSearch">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Search:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>txtSearch</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSearchClear">
        <property name="toolTip">
         <string>Clear Search bar</string>
        </property>
        <property name="icon">
         <iconset resource="klfres.qrc">
          <normaloff>:/pics/clearright.png</normaloff>:/pics/clearright.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtSearch">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Incremental search within history items. Find next with F3 key.</string>
        </property>
        <property name="whatsThis">
         <string>Incremental search within history items. Find next with F3 key.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnFindPrev">
        <property name="toolTip">
         <string>Find Previous [Shift-F3]</string>
        </property>
        <property name="whatsThis">
         <string>Find Previous occurrence of search string</string>
        </property>
        <property name="icon">
         <iconset resource="klfres.qrc">
          <normaloff>:/pics/findprev.png</normaloff>:/pics/findprev.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnFindNext">
        <property name="toolTip">
         <string>Find Next [F3]</string>
        </property>
        <property name="whatsThis">
         <string>Find Next occurrence of search string</string>
        </property>
        <property name="icon">
         <iconset resource="klfres.qrc">
          <normaloff>:/pics/findnext.png</normaloff>:/pics/findnext.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="frmButtons">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnRestore">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Restore current selected formula to KLatexFormula editor, with style</string>
        </property>
        <property name="text">
         <string>&amp;Restore Formula</string>
        </property>
        <property name="icon">
         <iconset resource="klfres.qrc">
          <normaloff>:/pics/restore.png</normaloff>:/pics/restore.png</iconset>
        </property>
        <property name="shortcut">
         <string>Alt+U</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDelete">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Delete de selected history items</string>
        </property>
        <property name="text">
         <string>&amp;Delete item(s)</string>
        </property>
        <property name="icon">
         <iconset resource="klfres.qrc">
          <normaloff>:/pics/delete.png</normaloff>:/pics/delete.png</iconset>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnImportExport">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Import/Export</string>
        </property>
        <property name="icon">
         <iconset resource="klfres.qrc">
          <normaloff>:/pics/importexport.png</normaloff>:/pics/importexport.png</iconset>
        </property>
        <property name="shortcut">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnConfig">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="icon">
         <iconset resource="klfres.qrc">
          <normaloff>:/pics/settings.png</normaloff>:/pics/settings.png</iconset>
        </property>
        <property name="shortcut">
         <string>Alt+N</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClose">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Close this window</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset resource="klfres.qrc">
          <normaloff>:/pics/closehide.png</normaloff>:/pics/closehide.png</iconset>
        </property>
        <property name="shortcut">
         <string>Alt+O</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>splitViewsEditor</zorder>
  <zorder>frmSearch</zorder>
  <zorder>frmButtons</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KLFLibEntryEditor</class>
   <extends>QWidget</extends>
   <header location="global">klflibentryeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnSearchClear</tabstop>
  <tabstop>txtSearch</tabstop>
  <tabstop>btnFindNext</tabstop>
  <tabstop>btnFindPrev</tabstop>
  <tabstop>btnRestore</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnImportExport</tabstop>
  <tabstop>btnConfig</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources>
  <include location="klfres.qrc"/>
 </resources>
 <connections/>
</ui>
